### ISSUE TITLE
Change customization failure message

### ISSUE THREAD

**Issue by murphysj** [ES-DOC/esdoc-questionnaire#226]
WHO: Sylvia

The current message in the pop up box says:

"Failed to save customizer."

It should say 

"Failed to save customization."

If we want to get fancy, we could say

"Failed to save customization $name."

**Comment by allynt** [comment]
Changes message in e7c32262eb9fa3b2d007fd4b7d7e23de2fcb984d.  Not using the "name" attribute in the message since an invalid name could have been part of the problem preventing the customization from saving.

**Comment by allynt** [comment]
If the name is invalid in both possible ways, the error messages are concatenated ungrammatically:

"Value may not contain spaces.value may not contain any of the following characters: '\ / < > % # % { } [ ] $ |'"

**Comment by allynt** [comment]
978590e34794e0b01cfbecdd37cb16f82d891971 addressed this, but not in the planned way.  The error in capitalization was fixed, but the error in concatenation was handled by adding some space between the messages.

The contatenation is actually the result of there being two validators that raised an exception.  Those get converted to an HTML list by Django Templates.  I have CSS to render that as a pretty red single line of text.  I have just adjusted that CSS.
